docker/docker/src/test/java/org/arquillian/cube/docker/impl/util/AutoStartOrderUtilTestCase.java

```diff
@@ -49,6 +49,28 @@
             "  links:\n" +
             "    - B:B\n";
 
+    private static final String SCENARIO_SINGLE_ROOT_AND_MULTIPLE_DEPENDS =
+            "A:\n" +
+            "  dependsOn:\n" +
+            "    - B\n" +
+            "B:\n" +
+            "  image: a\n" +
+            "C:\n" +
+            "  dependsOn:\n" +
+            "    - B\n";
+
+    private static final String SCENARIO_SINGLE_ROOT_AND_MULTIPLE_DEPENDS_AND_LINK =
+            "A:\n" +
+            "  dependsOn:\n" +
+            "    - B\n" +
+            "B:\n" +
+            "  image: a\n" +
+            "C:\n" +
+            "  links:\n" +
+            "    - A:A\n" +
+            "  dependsOn:\n" +
+            "    - B\n";
```

Comment:
Well we need to find all the places and change it since it is not only this class. Basically to have something common in all places and do not have both approaches.